pretest_clean.lua

```diff
@@ -25,9 +25,12 @@ local function clean()
     end):filter(function(name)
         -- skip internal spaces
         local first_char = string.sub(name, 1, 1)
-        return first_char ~= '_'
+        return (name ~= nil and first_char ~= '_')
```

Comment:
Can the name be nil? The same effect will be if name = {} (the filter will be passed), but this is also not a valid case.

Reply:
yep

Reply:
Right, changed.